ui/src/app/pages/analysis/reset-runtime-button.tsx

```diff
@@ -151,57 +151,57 @@ export class ResetClusterButton extends React.Component<Props, State> {
       <div style={styles.notebookSettings}>
         {this.createButtonAndLabel()}
       </div>
-      {this.state.resetClusterModal &&
+      {this.state.resetRuntimeModal &&
       <Modal data-test-id='reset-notebook-modal'
-             loading={this.state.resetClusterPending}>
+             loading={this.state.resetRuntimePending}>
         <ModalTitle>Reset Notebook Server?</ModalTitle>
         <ModalBody>
             <div>
               <strong>Warning:</strong> Any unsaved changes to your notebooks may be lost
-              and your cluster will be offline for 5-10 minutes.
+              and your runtime will be offline for 5-10 minutes.
```

Comment:
Is this PR meant to include user-facing changes? It feels like "server" may actually be more appropriate here, to be consistent with the modal title.

Reply:
Thanks, good call. You are correct that I did not intend for any user-visible changes here. I agree with "server" being better here, using that for completeness.